contractcourt/htlc_success_resolver.go

```diff
@@ -52,7 +52,7 @@ type htlcSuccessResolver struct {
 	// account any fees that may have to be paid if it goes on chain.
 	htlcAmt lnwire.MilliSatoshi
 
-	ResolverKit
+	contractResolverKit
```

Comment:
Is the pure `ResolverKit` used anywhere now after this change?

Reply:
yes, it is passed as an argument to all constructors

Reply:
has now been renamed to `ResolverConfig` to make its purpose clear